src/awst_build/eb/bytes-expression-builder.ts

```diff
@@ -273,6 +279,37 @@ export class BytesInvertBuilder extends ParameterlessFunctionBuilder {
   }
 }
 
+export class BitwiseOpFunctionBuilder extends FunctionBuilder {
+  constructor(
+    private expr: awst.Expression,
+    private op: BytesBinaryOperator,
+  ) {
+    super(expr.sourceLocation)
+  }
+
+  call(args: ReadonlyArray<NodeBuilder>, typeArgs: ReadonlyArray<PType>, sourceLocation: SourceLocation): NodeBuilder {
+    const {
+      args: [other],
+    } = parseFunctionArgs({
+      args,
+      typeArgs,
+      genericTypeArgs: 0,
+      callLocation: sourceLocation,
+      funcName: enumKeyFromValue(this.op, BytesBinaryOperator),
```

Comment:
you can just use
```suggestion
      funcName: BytesBinaryOperator[this.op],
```

Reply:
probably no need for the helper function in this case

Reply:
`BytesBinaryOperator[this.op]` throws `Element implicitly has an 'any' type because expression of type 'BytesBinaryOperator' can't be used to index type 'typeof BytesBinaryOperator'` error.